.github/workflows/format-check.yml

```diff
@@ -7,9 +7,22 @@ permissions: read-all
 
 on: [push, pull_request]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  SDE_MIRROR_ID: 813591
```

Comment:
Install SDE to run clang-format? No way.

Pulling our llvm distro is ok, but isn't it easier just to do `apt install clang-format-18`?

Reply:
> apt installs clang-format 18.1.3 and it has different formatting in comparison to latest 18.1.8.

😨 😨 😨